proposals/threads/Overview.md

```diff
@@ -437,6 +437,12 @@ For the web embedding, `atomic.notify` is equivalent in behavior to executing th
 1. Let `result` be [`Atomics.notify`][](`int32array`, `address`, `fcount`).
 1. Return `result` converted to an `i32`.
 
+## Fence operator
+
+The fence operator, `atomic.fence`, takes no operands, and returns nothing. It is intended to preserve the semantics of the [fence operators of higher-level languages](https://en.cppreference.com/w/cpp/atomic/atomic_thread_fence).
+
+Unlike other atomic operators, `atomic.fence` does not target a particular linear memory, and it is therefore *not* a validation error for this operator to be used in a module that does not declare a shared memory.
```

Comment:
nit: I assume it's also OK to be used in a module that declares no memory at all.

Reply:
yes; clarified this